tests/sentry/runner/commands/test_killswitches.py

```diff
@@ -11,17 +11,39 @@ class KillswitchesTest(CliTestCase):
 
     @mock.patch(
         "sentry.killswitches.ALL_KILLSWITCH_OPTIONS",
-        {OPTION: KillswitchInfo(description="hey", fields=("project_id", "event_type"))},
+        {
+            OPTION: KillswitchInfo(
+                description="hey", fields={"project_id": "hey", "event_type": "ho"}
```

Comment:
To avoid ambiguity of descriptions, can we replace one of hey's with something else?